app/celery/scheduled_tasks.py

```diff
@@ -332,6 +334,38 @@ def delete_dvla_response_files_older_than_seven_days():
         raise
 
 
+@notify_celery.task(name="raise-alert-if-letter-notifications-still-sending")
+@statsd(namespace="tasks")
+def raise_alert_if_letter_notifications_still_sending():
+    today = datetime.utcnow().date()
+
+    # Do nothing on the weekend
+    if today.isoweekday() in [6, 7]:
+        return
+
+    if today.isoweekday() == 1:
+        offset_days = 3
+    else:
+        offset_days = 1
+
+    still_sending = Notification.query.filter(
+        Notification.notification_type == LETTER_TYPE,
+        Notification.status == NOTIFICATION_SENDING,
+        Notification.created_at >= today - timedelta(days=offset_days),
+        Notification.created_at < today
+    ).count()
```

Comment:
How about notifications that are created after 5:40pm (the cut-off time) yesterday?

Reply:
I've asked @servingUpAces about this and she suggested changing `created_at` to `sent_at` since that's set when the file is sent to DVLA, so it will be the correct date both for letters created on the day and for letters created after 5:30pm the day before. I've updated the task and the tests to do this instead.